cli/src/commands/whoami.rs

```diff
@@ -220,6 +220,13 @@ pub async fn run(State(env): State<CliEnv>) {
                 c_error!("Admin Service '{}' is unhealthy:", client.base_url);
                 c_eprintln!("   >> {}", e);
             }
+        },
+        Err(e) => {
+            c_error!(
+                "Maybe cannot connect to Admin Service '{}':",
+                &env.admin_base_url().unwrap().clone()
```

Comment:
`env.admin_base_url()` might return an error if no admin base url has been specified. That's why we shouldn't `unwrap` here. Maybe it's enough to only log the error here.